modules/fb_instant_articles_display/src/EntityPropertyMapper.php

```diff
@@ -109,16 +109,27 @@ private function addHeaderFromProperties() {
       );
     }
 
+    // user token or static string for author name
+    $author = '';
+    if (module_exists('Token')) {
+      $display_author = variable_get('fb_instant_articles_display_author', '[node:author]');
+      $author = token_replace($author_token, array('node' => $node));
+    } else {
+      $author = variable_get('fb_instant_articles_display_author', '');
+    }
     // Default the article author to the username.
     if (isset($this->entity->uid)) {
-      $author = user_load($this->entity->uid);
-      if ($author) {
-        $header->addAuthor(
-          Author::create()
-            ->withName($author->name)
-        );
+      if (empty($author)) {
+        $display_author = user_load($this->entity->uid);
+        $author = $display_author->name;
       }
     }
+    if ($author) {
```

Comment:
If you check `if (!empty($author)) {` you don't need to define the empty string `$author = '';` above on line 113 (cleaner not to do that if you don't need to).